### ISSUE TITLE
BD is not working after Discord update - using Linux

### ISSUE THREAD

**Issue by fhuewhd** [bb010g/betterdiscordctl#14]
Today I loaded Discord for the first time since at least the end of April and it insisted on installing an update before it would load. After installing the update and trying again, I found BD was gone and is not in the Settings.

I originally installed BD using Betterdiscordctl (BandagedBD doesn't appear to have a Linux installer of its own), so I tried using that to reinstall BD, but only got "ERROR: Discord installation not found.", which is ridiculous.  I've tried "upgrade", which worked, or said it did, although there were a lot of quibbles after it, and "install" and "uninstall" both of which gave the error.

(Here are the quibbles I saw when I did "upgrade", just in case they're relevant:)
ab@ab-Satellite:~$ betterdiscordctl upgrade
Downloading betterdiscordctl...
Successfully updated betterdiscordctl.
/usr/local/bin/betterdiscordctl: line 587: local: can only be used in a function
/usr/local/bin/betterdiscordctl: line 589: local: can only be used in a function
/usr/local/bin/betterdiscordctl: line 590: local: can only be used in a function
/usr/local/bin/betterdiscordctl: line 591: local: can only be used in a function
/usr/local/bin/betterdiscordctl: line 592: local: can only be used in a function
/usr/local/bin/betterdiscordctl: line 497: =MAJOR_A: command not found
/usr/local/bin/betterdiscordctl: line 497: =MAJOR_B: command not found
/usr/local/bin/betterdiscordctl: line 597: return: can only `return' from a function or sourced script
/usr/local/bin/betterdiscordctl: line 607: return: can only `return' from a function or sourced script
/usr/local/bin/betterdiscordctl: line 609: return: can only `return' from a function or sourced script
/usr/local/bin/betterdiscordctl: line 610: syntax error near unexpected token `}'
/usr/local/bin/betterdiscordctl: line 610: `}'

I'm using Zorin 8 Linux, a variation of Ubuntu Trusty Tahr, and BDCTL says it's GitHub version 1.2.1 and accordinhg to Ubuntu Software Centre the version of Discord I've got is 0.0.5.

Any ideas please? I can't go to the support server, as I have sensory issues and can't cope with Discord without BD to get rid of the animations!

**Comment by ObserverOfTime** [comment]
The reported line numbers don't match up to the source code. Have you edited BDCTL in any way?

*Side note: `upgrade` could use a makeover.*

Can you rerun `uninstall` and `install` with double verbosity (`-v -v`) and post the output here?

**Comment by fhuewhd** [comment]
I certainly have not edited BDCTL, I wouldn't know how.  So I don't know what happened there.
But somebody on the BD issues page came up with a solution:

> by default, betterdiscordctl scans /opt for any discord install, and from what i've seen, some installation goes to /usr/share and you could change the scan directory by using -s [directory]
> 
> p.s. betterdiscordctl is like the semi official installer for bandagedbd

I tried that and it did the trick - BDCTL did its thing correctly as it did when I first used it, and BD is now working fine and dandy.  Thanks for your help.

Sorry, I forgot to try the verbosity thing and don't like to disturb BD again now, when I've got it working!
